src/services/services.ts

```diff
@@ -7668,6 +7681,22 @@ namespace ts {
             return [];
         }
 
+        function getCodeFixesAtPosition(fileName: string, start: number, end: number, errorCodes: string[]): CodeFix[] {
+            synchronizeHostData();
+            const sourceFile = getValidSourceFile(fileName);
+
+            let fixes: CodeFix[] = [];
+
+            errorCodes.forEach(error => {

```

Comment:
can we use ES5 methods like `forEach` in services? Regardless, this looks more like a `map` -- or at least the loop body should use `fixes.push(fix)`